src/main/java/com/internet/shop/controller/user/UserRegistrationController.java

```diff
@@ -31,8 +31,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         if (password.equals(req.getParameter("repeatPwd"))) {
             User createdUser = userService.create(new User(login, password));
             cartService.create(new ShoppingCart(createdUser.getId()));
-            req.setAttribute("login", createdUser.getLogin().toUpperCase());
-            req.getRequestDispatcher("/WEB-INF/views/user/userMainPage.jsp").forward(req, resp);
+            resp.sendRedirect(req.getContextPath() + "/login");
```

Comment:
user has only created his account, should he pass his data once again?

Reply:
done